.github/workflows/test_pr.yml

```diff
@@ -24,6 +24,39 @@ jobs:
       - name: Black
         run: python -m black --diff --check .
 
+  convention_check:
+    name: Convention check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cycodelabs/cimon-action@v0
+        with:
+          prevent: true
+          fail-on-error: true
+      
+      - name: Check branch name
+        env:
+          BRANCH_NAME: ${{ github.head_ref }}
+        run: |
+          branch_prefix="^(feat|fix|refactor|chore|docs|test|library)\/[a-z0-9._-]+$"
```

Comment:
What about hotfix?
Also - I prefer "feature" over "feat" but I will leave it for you to decide

Reply:
Changed according to: https://dev.to/varbsan/a-simplified-convention-for-naming-branches-and-commits-in-git-il4